tests/markdown.js

```diff
@@ -0,0 +1,150 @@
+/*jshint multistr: true */
```

Comment:
yeah. I just copied this file from javaRosa so I didn't need to do the boilerplate. cleaned this up